src/AbstractUpdateAction.php

```diff
@@ -19,6 +19,8 @@ class AbstractUpdateAction extends Param
 
     /**
      * Sets the id of the document.
+     *
+     * @return static
```

Comment:
```suggestion
     * @return $this
```

That's the way this kind of return is type-hinted in the codebase.

Reply:
👍 changed